src/simple_websocket/ws.py

```diff
@@ -397,7 +397,7 @@ def __init__(self, url, subprotocols=None, headers=None,
         elif isinstance(headers, list):
             self.extra_headeers = headers
 
-        sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```

Comment:
Will this allow the client to connect to a IPv4 only server? I'm not really sure this is true for every platform, especially on Windows. I would prefer an option passed in the constructor to select IPv6.

Reply:
@miguelgrinberg i tested on MacOS and on Linux and both work for IPV4 and IPV6 but didn't tried on Windows, indeed. I just changed my last code to use `address_family` parameter in the class constructor and use the default as `socket.AF_INET` so no break changes in the behavior.

Thanks for the feedback!